### PR TITLE
RBAC functionalities

### PR DESCRIPTION
**Pull request by abilpraju-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE
https://aottech.atlassian.net/browse/FWF-1907
https://aottech.atlassian.net/browse/FWF-2125
image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

**Review by abhilash-aot — APPROVED** [review]

File: forms-flow-web/src/modules/authorizationReducer.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import ACTION_CONSTANTS from "../actions/actionConstants";
+
+// reducers.js
+export const initialState = {
+  roles: [],
+  userError: null,
```
Comment (shuhaib-aot) [inline forms-flow-web/src/modules/authorizationReducer.js:6]: is this variable contain only userError?

File: forms-flow-web/src/styles.scss [hunk 2]
```diff
@@ -817,4 +817,35 @@ div.upload input[type="file"] {
...
+      color: #393939;
+      font-weight: 100;
+      cursor: pointer;
+      display: inline-flex;
+      text-align: center;
+      align-items: center;
+      justify-content: center;
+      
+      &:hover{
+          background-color: #fc4f4f;
```
Comment (shuhaib-aot) [inline forms-flow-web/src/styles.scss:847]: could you please check the color variable name is there in our code

**Review by shuhaib-aot — CHANGES_REQUESTED** [review]
Please resolve the conversations

**Comment by abhilash-aot** [comment]
Point to develop branch and rebase to develop

**Review by abhilash-aot — APPROVED** [review]

**Review by AbijithS-aot — APPROVED** [review]

**Pull request merged**